packages/zowe-explorer/README.md

```diff
@@ -83,7 +89,33 @@ Before using the base profile functionality, ensure that you have [Zowe CLI](htt
 
 For more information, see [Integrating with API Mediation Layer](https://docs.zowe.org/stable/user-guide/cli-usingcli.html#integrating-with-api-mediation-layer).
 
-### Sample use cases
+#### Log in to the Authentication Service
+
+Use the Log in to the **Authentication Service** feature to regenerate a new token for your base profile.
+
+**Follow these steps:**
+
+1. Open Zowe Explorer.
+2. Right-click your profile.
+3. Select the **Log in to Authentication Service** option.
+
+   You will be prompted to enter your username and password beforehand.
```

Comment:
I think you can delete "beforehand" since you are prompted after you select the option.